coffee/session.coffee

```diff
@@ -136,21 +160,10 @@ class palava.Session extends @EventEmitter
     @room.on 'signaling_not_reachable',     => @emit 'signaling_not_reachable'
     true
 
-  # Reconnect the session
-  #
-  reconnect: =>
-    @emit 'session_reconnect'
-    @destroy(true)
-    @resetOptions(@initOptions, true)
-    @init({}, true)
-
   # Destroys the session
-  #
-  # @param reconnect  [Boolean] Is part of a reconnect?
-  #
-  destroy: (reconnect = false) =>
-    @emit 'session_before_destroy' unless reconnect
+  destroy: =>
+    @emit 'session_before_destroy'
```

Comment:
why was this split into before and after again? do we still need that?

Reply:
My reasoning is as follows: Someone interested in running code when the session is closing down can have different use cases, which vary a lot. If you, for example, might want to take a photo using the user media, you need the before callback. If you want to sent a tracking event that the session has been ended, you rather choose the after callback.

Reply:
Ok, the usage examples make sense!